test/e2e/README.md

```diff
@@ -1,85 +0,0 @@
-<!--
-* Licensed to the Apache Software Foundation (ASF) under one
-* or more contributor license agreements.  See the NOTICE file
-* distributed with this work for additional information
-* regarding copyright ownership.  The ASF licenses this file
-* to you under the Apache License, Version 2.0 (the
-* "License"); you may not use this file except in compliance
-* with the License.  You may obtain a copy of the License at
-*
-*      http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
--->
-
-# End to End Testing in YuniKorn-K8shim
```

Comment:
How about removing all the content but put a link to developer guide? @chia7712  WDYT?

# End to End Testing in YuniKorn-K8shim

This document has been moved to XXXX.

Reply:
+1

In fact, I always take a look at `README` first for docs :)

Reply:
Nice idea. 
Done.